packages/cashier/src/Components/Form/__tests__/account-transfer-form.spec.js

```diff
@@ -0,0 +1,451 @@
+                value: 'value',
+            },
+        ],
+        account_limits: {
+            daily_transfers: {
+                dxtrade: {},
+                internal: {},
+                mt5: {},
+            },
+        },
+        minimum_fee: '0',
+        mt5_login_list: [
+            {
+                login: 'value',
+                market_type: 'gaming',
+                server_info: {
+                    geolocation: {
+                        region: 'region',
+                        sequence: 0,
+                    },
+                },
+            },
+        ],
+        selected_from: { currency: 'USD', is_mt: false },
+        selected_to: { currency: 'USD', is_mt: false },
+        transfer_fee: 2,
+        transfer_limit: {
+            min: 0,
+            max: 1000,
+        },
+        onMount: jest.fn(),
+        resetConverter: jest.fn(),
+        recentTransactionOnMount: jest.fn(),
+        requestTransferBetweenAccounts: jest.fn(),
+    });
+
+    it('component should be rendered', () => {
+        const props = mockProps();
+
+        const { container } = render(<AccountTransferForm {...props} />);
+
+        expect(container.querySelector('.account-transfer-form__wrapper')).toBeInTheDocument();
+        expect(screen.getByText('Transfer between your accounts in Deriv')).toBeInTheDocument();
+    });
+
+    it('should show loader if account_list.length === 0', () => {
+        const props = mockProps();
+        props.accounts_list = [];
+
+        const { container } = render(<AccountTransferForm {...props} />);
+
+        expect(container.querySelector('.cashier__loader-wrapper')).toBeInTheDocument();
+    });
+
+    it('should show <Form /> component if account_list.length > 0', () => {
+        const props = mockProps();
+
+        const { container } = render(<AccountTransferForm {...props} />);
+
+        expect(screen.getByText('From')).toBeInTheDocument();
+        expect(screen.getByText('To')).toBeInTheDocument();
+        expect(container.querySelector('.account-transfer-form__drop-down-wrapper')).toBeInTheDocument();
+        expect(container.querySelector('.account-transfer-form__drop-down')).toBeInTheDocument();
+        expect(container.querySelector('.account-transfer-form__drop-down--to-dropdown')).toBeInTheDocument();
+        expect(container.querySelector('.account-transfer-form__form-submit')).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: 'Transfer' })).toBeInTheDocument();
+    });
+
+    it('should show an error if amount is not provided', async () => {
+        const props = mockProps();
+        props.setErrorMessage = jest.fn();
+        props.setAccountTransferAmount = jest.fn();
+
+        const { container } = render(<AccountTransferForm {...props} />);
+
+        const amount_field = container.querySelector('input[name=amount]');
+        const submit_button = screen.getByRole('button', { name: 'Transfer' });
+
+        act(() => {
```

Comment:
@artsemkavaleu-deriv What are the need for these `act`s, `fireEvent` is already wrapped in `act` itself.